net-misc/i2pd/i2pd-2.6.0-r3.ebuild

```diff
@@ -28,8 +28,8 @@ DEPEND="${RDEPEND}
 	i2p-hardening? ( >=sys-devel/gcc-4.7 )
 	|| ( >=sys-devel/gcc-4.7 >=sys-devel/clang-3.3 )"
 
-I2PD_USER="${I2PD_USER:-i2pd}"
-I2PD_GROUP="${I2PD_GROUP:-i2pd}"
+I2PD_USER=i2pd
+I2PD_GROUP=i2pd
 
```

Comment:
It seemed strange to depend on the environment variables here. I looked at ebuilds that create users: lighttpd, apache, openssh, transmission, smokeping - they all create users and use fixed names for that, not depending on environment variables. But I am not 100% percent sure about this change and I would be happy to revert it back. Maybe @tomboy-64 knows?